### ISSUE TITLE
Headers on summary coded as paragraphs

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#829]
### Description of the bug

The header "Endring til navn" in the example under here is coded as a paragraph, but styled as a header. Everything needs to be coded as they appear.

![Skjermbilde 2023-01-11 kl  13 52 34](https://user-images.githubusercontent.com/33222679/211811488-e7a3ec29-c9e2-4079-b848-b2332fc5d789.png)

### Steps To Reproduce

1. Go to a summary page with a header
2. Check if it is `<p>` or `<h1-5>`

**Comment by bjosttveit** [comment]
It appears that `DisplayGroupContainer` uses a `<p>` element with css to make it bigger:
```jsx
<Typography
  className={classes.groupTitle}
  variant='body1'
>
  {title}
</Typography>
```

Not sure exactly what it should be changed to. It is similar in size to `h2` elements, so maybe that?

**Comment by bjosttveit** [comment]
What do you think is appropriate in this case @Febakke ? A quick fix would be to set it as a `h2`-element, as there is no guarantee that it appears under another heading element. Another option is to make it customizable with (`L`, `M`, `S`) like the `Header` component is.
